TombEngine/Math/Geometry.h

```diff
@@ -1,9 +1,9 @@
 #pragma once
 
-class AxisAngle;
-class EulerAngles;
-class Pose;
-class Vector3i;
+namespace TEN::Math { class AxisAngle; };
+namespace TEN::Math { class EulerAngles; };
+namespace TEN::Math { class Pose; };
+namespace TEN::Math { class Vector3i; };
 
```

Comment:
Shall it repeated namespaces or combine all under a single one?

```c++
namespace TEN::Math
{
    class AxisAngle; 
    class EulerAngles;
    class Pose;
    class Vector3i; 
};
```

Reply:
Probably not, it might be repetitive but if there are multiple namespaces headers get messy.